js/utils.js

```diff
@@ -8,7 +8,32 @@ require.scopes.utils = ( () => {
         return host;
     }
 
+    function syncToStorage(data){
+        chrome.storage.local.set(data, function() {
+            console.log("updated local storage data");
+        });
+    }
+
+    function getFromStorage(key, callback){
+        chrome.storage.local.get(key, function(result){
+            if(result[key]){
+                callback(result[key]);
+            }
+        });
+    }
+
+    function getCurrentURL(callback){
```

Comment:
Ok. Maybe we need something different for the popup then. I'm not sure why it doesn't work from the popup but it's working in the background here:
https://github.com/duckduckgo/chrome-zeroclickinfo/blob/b2b8e801a8ff9cef74950e93e945c199787f0ba0/js/background.js#L285